non-protocol-specs/0013-NP-POSE-position-estimate.md

```diff
@@ -46,6 +46,10 @@ The position estimate request has an additional `include_collateral_increase_in_
 
 The endpoint request contains additional optional argument `scale_liquidation_price_to_market_decimals`. When set to `false` the liquidation price estimates are scaled to asset decimal places, when set to `true` these estimates are scaled to market decimal places.
 
+### Price cap
+
+When a price cap is specified it should be assumed that the estimate is to be provided for a [capped futures](./../protocol/0016-PFUT-product_builtin_future.md#1-product-parameters) market. The liquidation price estimate should not be returned in that case and margin levels as well collateral increase estimate should be as per fully-collateralised margin [spec](./../protocol/0019-MCAL-margin_calculator.md#fully-collateralised).
```

Comment:
Surely we would expect a liquidation price estimate if there's a cap but not a fully collateralised flag?

Reply:
Good point, fixed now